src/allencell_ml_segmenter/main/main_widget.py

```diff
@@ -83,7 +84,7 @@ def __init__(
         self._training_model: TrainingModel = TrainingModel(
             main_model=self._model, experiments_model=self._experiments_model
         )
-
+        self._file_input_model: FileInputModel = FileInputModel()
```

Comment:
Why are we initializing the `FileInputModel` here for prediction, but not for postprocessing?